receiver/sqlqueryreceiver/logs_receiver.go

```diff
@@ -286,19 +286,19 @@ func (queryReceiver *logsQueryReceiver) collect(ctx context.Context) (plog.Logs,
 		return logs, fmt.Errorf("error getting rows: %w", err)
 	}
 
-	var errs error
+	var errs []error
 	scopeLogs := logs.ResourceLogs().AppendEmpty().ScopeLogs().AppendEmpty().LogRecords()
 	for logsConfigIndex, logsConfig := range queryReceiver.query.Logs {
 		for _, row := range rows {
 			logRecord := scopeLogs.AppendEmpty()
 			rowToLog(row, logsConfig, logRecord)
 			logRecord.SetObservedTimestamp(observedAt)
 			if logsConfigIndex == 0 {
-				errs = multierr.Append(errs, queryReceiver.storeTrackingValue(ctx, row))
+				errs = append(errs, queryReceiver.storeTrackingValue(ctx, row))
 			}
 		}
 	}
-	return logs, nil
+	return logs, errors.Join(errs...)
```

Comment:
`errs` was originally being put together in the above loop, but not returned. I assume this was a bug and `errs` should be returned.